working_group/2024/2024-03-07-meeting-notes.md

```diff
@@ -0,0 +1,31 @@
+# 2024 March 7 - Meeting notes
+
+## Meeting recording
+
+Watch the YouTube recording of the meeting on the Salt Extensions playlist: https://www.youtube.com/playlist?list=PL9svBjLDUl_-gyXj2lDYomI8keXtbMqUl
+
+## Agenda and notes
+
+- **Welcome any newcomers**
+- **Announcements**
+  - Reminder that our new meeting cadence is the 1st and 3rd Thursday. See the [Salt Project community calendar](https://saltproject.io/calendar/) for specific dates and the Zoom link.
+- **Update from Alyssa from engineering**
+  - After our last meeting, Alyssa took some of the working group's concerns to the engineers as well as the professional services and support teams at Broadcom for feedback. She'll share some of the outcomes of those discussions with the group.
+- **Feedback to the core Salt team about the modules**
+  - Any concerns about this list that we need to raise with the core Salt team?
+- **Discussion about Max's proposal to keep centralized documentation**
+  - See [Keep the centralized module documentation at docs.saltproject.io](https://github.com/saltstack/salt/discussions/66144) on GitHub discussions.
+- **Next steps**

```

Comment:
**suggestion (non-blocking):** add an agenda item to update on salt-pip bugfix (line 23 from working_group/2024/2024-02-15-meeting-notes.md) via this [relenv PR](https://github.com/saltstack/relenv/pull/173)

Reply:
Cool! I added that to the agenda.